cmd/config.go

```diff
@@ -0,0 +1,37 @@
+package cmd
+
+import (
+	"io"
+	"os"
+
+	cmdUtil "github.com/flyteorg/flytectl/pkg/commandutils"
+	"github.com/flyteorg/flytectl/pkg/docker"
+	f "github.com/flyteorg/flytectl/pkg/filesystemutils"
+	"github.com/flyteorg/flytectl/pkg/util"
+	"github.com/spf13/cobra"
+)
+
+// configCmd represents the config init command
+var configCmd = &cobra.Command{
+	Use:   "init-config",
+	Short: "init-config flytectl config",
+	Long: `
+init-config will create flytectl config in flyte directory i.e ~/.flyte/config.yaml 
+::
+
+ bin/flytectl init-config
+Usage`,
+	RunE: func(cmd *cobra.Command, args []string) error {
+		return initFlytectlConfig(os.Stdin)
+	},
+}
+
+func initFlytectlConfig(reader io.Reader) error {
+	if err := docker.SetupFlyteDir(); err != nil {
+		return err
+	}
+	if cmdUtil.AskForConfirmation("Are you sure ? It will overwrite the default config from ~/.flyte/config.yaml", reader) {
+		return util.WriteIntoFile([]byte(util.ConfigTemplate), f.FilePathJoin(f.UserHomeDir(), ".flyte", "config.yaml"))
```

Comment:
The goal of this command is to allow users to setup config given an endpoint...
examples:

```
# Setup default config to sandbox
flytectl init config

# Setup config that talks to flyte.mycompany.com
flytectl init config --host flyte.mycompany.com

# Setup config with custom storage config
flytectl init config --host flyte.mycompany.com --storage.type=s3 --storage.container=my-bucket 
```